src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTag.cs

```diff
@@ -87,6 +90,9 @@ public override GraphicsResult GetGraphics(IWpfTextView view, Geometry bounds, T
                 Padding = new Thickness(1)
             };
 
+            var editorBackground = (Color)_editorFormatMap.GetProperties("Indicator Margin")["BackgroundColor"];
```

Comment:
Since "Indicator Margin" could potentially be themed separately from the Editor background should we be using it here?

Reply:
I guess not, is there some other identifier that I should be using to get the editor background?